### PR TITLE
feat: improve relay modal and add alias functionality

### PR DESCRIPTION
**Pull request by 0xArdy**
# 🤖 Linear

closes FRA-30
closes FRA-31

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -60,6 +60,10 @@ export const App = () => {
                 fontSize: '62.5%',
                 backgroundColor: currentTheme.backgroundSecondary,
               },
+              '&.MuiInputBase-root.MuiInputBase-colorPrimary.MuiInputBase-fullWidth.MuiInputBase-adornedEnd:hover': {
```
Comment (turtlemoji) [inline src/App.tsx:63]: I don't know if this the best way, looks weird to have this styles in here

File: src/hooks/useVault.tsx [hunk 2]
```diff
@@ -57,6 +59,15 @@ export const useVault = ({
         const writeResult = await writeAsync();
         await publicClient.waitForTransactionReceipt(writeResult);
         setModalOpen(ModalType.NONE);
+
+        // Update alias
+        if (newAliasData) {
+          const temp = { ...aliasData, ...newAliasData };
+          console.log(aliasData, temp);
```
Comment (turtlemoji) [inline src/hooks/useVault.tsx:66]: remove later

Comment (0xArdy) [reply src/hooks/useVault.tsx]: done https://github.com/defi-wonderland/keep3r-framework-app/pull/2/commits/9d3dbf8df6852bf186a8e00930f469931a41b9b0

File: src/App.tsx (hunk 1 above)
Comment (0xArdy) [reply src/App.tsx]: fixed https://github.com/defi-wonderland/keep3r-framework-app/pull/2/commits/dff768ca7bc85c1a3d40badca90d0b0b81e10ca3

**Review by turtlemoji — APPROVED** [review]

**Pull request merged**